### PR TITLE
6776 remove unused routes

### PR DESCRIPTION
**Pull request by denys281**
@theworldbright 

PR for https://github.com/diaspora/diaspora/issues/6776

### CODE REVIEW

**Comment by denschub** [comment]
Have you checked if there are any other invalid routes?

**Comment by denys281** [comment]
@denschub  no, but I will check.

**Comment by denschub** [comment]
How did your checks go?

**Comment by denys281** [comment]
@denschub just started. Create docs for invalid routes: https://docs.google.com/spreadsheets/d/1rX6KRalSBKGm-aZrqSgDnO-xXmnNI-TGTSK5KybS8vI/edit?usp=sharing

But I don't now application well so maybe some of them aren't invalid. So usually it is resources in routes but now action in controller.

**Comment by denys281** [comment]
@denschub  Finished with document. What are the next steps?

**Comment by denschub** [comment]
How about... removing them? :)

**Comment by denys281** [comment]
Ok :)

File: config/routes.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 Diaspora::Application.routes.draw do
 
-  resources :report, :except => [:edit, :new]
+  resources :report, :except => [:edit, :new, :show]
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

File: config/routes.rb [hunk 2]
```diff
@@ -25,11 +25,11 @@
 
   get 'oembed' => 'posts#oembed', :as => 'oembed'
   # Posting and Reading
-  resources :reshares
+  resources :reshares, :only => [:create]
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

File: config/routes.rb [hunk 3]
```diff
@@ -25,11 +25,11 @@
...
 
   resources :status_messages, :only => [:new, :create]
 
-  resources :posts do
+  resources :posts, :only => [:show, :destroy] do
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

File: config/routes.rb [hunk 4]
```diff
@@ -57,7 +57,7 @@
   get "commented" => "streams#commented", :as => "commented_stream"
   get "aspects" => "streams#aspects", :as => "aspects_stream"
 
-  resources :aspects do
+  resources :aspects, :except => [:index, :new, :edit] do
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

File: config/routes.rb [hunk 5]
```diff
@@ -67,15 +67,15 @@
 
   get 'bookmarklet' => 'status_messages#bookmarklet'
 
-  resources :photos, :except => [:index, :show] do
+  resources :photos, :only => [:destroy, :create] do
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

File: config/routes.rb [hunk 6]
```diff
@@ -67,15 +67,15 @@
...
     put :make_profile_photo
   end
 
 	#Search
 	get 'search' => "search#search"
 
-  resources :conversations do
-    resources :messages, :only => [:create, :show]
+  resources :conversations, :except => [:edit, :update, :destroy]  do
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

Comment (diaspora-code-review) [inline config/routes.rb]: Unnecessary spacing detected.

File: config/routes.rb [hunk 7]
```diff
@@ -67,15 +67,15 @@
...
+    resources :messages, :only => [:create]
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

File: config/routes.rb [hunk 8]
```diff
@@ -156,30 +154,24 @@
   resources :profiles, :only => [:show]
 
 
-  resources :contacts,           :except => [:update, :create] do
-  end
+  resources :contacts, :only => [:index]
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

File: config/routes.rb [hunk 9]
```diff
@@ -156,30 +154,24 @@
...
   resources :share_visibilities,  :only => [:update]
   resources :blocks, :only => [:create, :destroy]
 
   get 'i/:id' => 'invitation_codes#show', :as => 'invite_code'
 
   get 'people/refresh_search' => "people#refresh_search"
-  resources :people, :except => [:edit, :update] do
-    resources :status_messages
-    resources :photos
+  resources :people, :only => [:show, :index] do
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

File: config/routes.rb [hunk 10]
```diff
@@ -156,30 +154,24 @@
...
+    resources :status_messages, :only => [:new, :create]
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

File: config/routes.rb [hunk 11]
```diff
@@ -156,30 +154,24 @@
...
+    resources :photos, :except => [:new, :update]
```
Comment (diaspora-code-review) [inline config/routes.rb]: Use the new Ruby 1.9 hash syntax.

Comment (diaspora-code-review) [inline config/routes.rb]: Use %i or %I for array of symbols.

**Comment by denys281** [comment]
@denschub ready.

**Comment by cmrd-senya** [comment]
The contact action `contacts#show` is likely to be implemented in #6120, since the aspect membership [rendering](https://github.com/diaspora/diaspora/blob/develop/app/controllers/people_controller.rb#L159-L170) is being moved to frontend, and therefore we need a way to fetch contact data from pod along with memberships.

**Comment by jhass** [comment]
Merged as ef729425b44e9f252794fd0a627d72cfdf286440

Sorry for taking so long to take a look at this.

Thank you!

**Pull request closed without merging**